docs/adding-new-platforms.md

```diff
@@ -0,0 +1,81 @@
+## Adding new platforms
+
+Depending on what type of platform you're adding, the instructions here will
+vary.
+
+### Adding a brand new platform
+
+- Create a new file for your mainboard in `platform-configs/`. For most
+  platforms this file will be called `[platform-vendor]-[platform-model].robot`.
+- Add the following at the top of your platform config:
+
+```robot
+*** Settings ***
+Resource    default.robot
+```
+
+- Copy the contents of `include/default.robot` to your platform config
```

Comment:
@mkopec I'm not sure I understand, by `your platform config` you mean `[platform-vendor]-[platform-model].robot` that I just created?

Reply:
IIUC right now it is added by e.g. `Resource include/pcengines.robot`

Reply:
> I'm not sure I understand, by your platform config you mean [platform-vendor]-[platform-model].robot that I just created?

Exactly, copy the contents of the file. We will be changing some of the settings and removing those we don't need.
Maybe that's not the easiest or best way to do this, I'll rethink this paragraph

Reply:
@mkopec understood, but please consider improving phrasing because it may be confusing.

Reply:
Tried to improve the phrasing

Reply:
Just for reference 39a46e493b894de0077a835a3dd0ddc411c2fe6f

Thanks